src/Stratis.Bitcoin.Features.PoA/PoABlockDefinition.cs

```diff
@@ -19,8 +19,9 @@ public class PoABlockDefinition : BlockDefinition
             ILoggerFactory loggerFactory,
             ITxMempool mempool,
             MempoolSchedulerLock mempoolLock,
-            Network network)
-            : base(consensusManager, dateTimeProvider, loggerFactory, mempool, mempoolLock, new MinerSettings(NodeSettings.Default(network)), network)
+            Network network,
+            NodeSettings nodeSettings)
+            : base(consensusManager, dateTimeProvider, loggerFactory, mempool, mempoolLock, new MinerSettings(nodeSettings), network)
```

Comment:
Are we not injecting the settings? `MinerSettings`

Reply:
No, nowhere in the codebase it's injected